cranelift/codegen/src/loop_analysis.rs

```diff
@@ -190,6 +190,19 @@ impl LoopAnalysis {
         self.valid = false;
     }
 
+    // Determines if a block dominates any predecessor
+    // and thus is a loop header
+    fn check_loop_header(
+        block: Block,
+        cfg: &ControlFlowGraph,
+        domtree: &DominatorTree,
+        layout: &Layout,
+    ) -> bool {
+        // loop header if it dominates any of its predecessors
```

Comment:
Full sentence is preferable: "A block is a loop header if it dominates any of its predecessors."